Makefile

```diff
@@ -101,8 +101,8 @@ tsb_mp:  ## Deploys MP
 		terraform output ${terraform_output_args} | jq . > ../../outputs/terraform_outputs/terraform-tsb-mp.json; \
 		fqdn=`jq -r '.tsb_fqdn' ../../terraform.tfvars.json`; \
 		address=`jq -r "if .ingress_ip.value != \"\" then .ingress_ip.value else .ingress_hostname.value end" ../../outputs/terraform_outputs/terraform-tsb-mp.json`; \
-		terraform -chdir=../../modules/$$cloud/register_fqdn init; \
-		terraform -chdir=../../modules/$$cloud/register_fqdn apply ${terraform_apply_args} -var=address=$$address -var=fqdn=$$fqdn; \
+		terraform -chdir=../fqdn/$$cloud init; \
+		terraform -chdir=../fqdn/$$cloud apply ${terraform_apply_args} -var-file="../../../terraform.tfvars.json" -var=address=$$address -var=fqdn=$$fqdn; \
```

Comment:
This module doesn't need all the variables, that's why I was not passing them. Why do we need to pass the entire file?
Also, the path looks wrong, shouldn't it be `../../terraform.tfvars.json`?

Reply:
as I have switched to the terraform_remote_state instead of approaching it with jsondecode and sub-local vars, there is no impact on the performance here, but it is a manageable consistency

Reply:
in regards to the path, it requires triple ```../../..``` as terraform.tfvars.json is three levels above the tsb/fqdn/$$cloud 
... 
I am doing manual tests of the PR... so far looks good, however I have not tested the private DNS